apps/api/src/persistence/migrations/04/migrate-bilingual-songs.migration.ts

```diff
@@ -0,0 +1,69 @@
+import { LanguageCode } from '@coscrad/api-interfaces';
+import { buildMultilingualTextFromBilingualText } from '../../../domain/common/build-multilingual-text-from-bilingual-text';
+import { MultilingualText } from '../../../domain/common/entities/multilingual-text';
+import { Song } from '../../../domain/models/song/song.entity';
+import { DTO } from '../../../types/DTO';
+import { ArangoCollectionId } from '../../database/collection-references/ArangoCollectionId';
+import { ArangoDatabaseDocument } from '../../database/utilities/mapEntityDTOToDatabaseDTO';
+import { ICoscradMigration } from '../coscrad-migration.interface';
+import { ICoscradQueryRunner } from '../coscrad-query-runner.interface';
+
+type SongDocument = ArangoDatabaseDocument<DTO<Song>>;
+
+type OldSongDocument = Omit<SongDocument, 'title'> & {
+    title: string;
+    titleEnglish: string;
+};
+
+const buildMultilingualText = (title: string, titleEnglish: string): DTO<MultilingualText> => {
+    return buildMultilingualTextFromBilingualText(
+        { text: title, languageCode: LanguageCode.Chilcotin },
+        {
+            text: titleEnglish,
+            languageCode: LanguageCode.English,
+        }
+    ).toDTO();
+};
+
+const extract = (
+    languageCodeToExtract: LanguageCode,
+    multilingualTextDto: DTO<MultilingualText>
+): string | null =>
+    multilingualTextDto.items.find(({ languageCode }) => languageCode === languageCodeToExtract)
+        ?.text || null;
+
+export class MigrateBilingualSongs implements ICoscradMigration {
+    sequenceNumber = 4;
+
+    name = `MigrateBilignualSongs`;
```

Comment:
(sp) MigrateBilignualSongs

Reply:
ah! I'll need to fix this in TNG's database manually I suppose as well. Good catch!

Reply:
fixed